knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -11,19 +10,59 @@
 from openpyxl import load_workbook
 from openpyxl.cell import Cell
 from openpyxl.worksheet.worksheet import Worksheet
+from knora.dsplib.models.helpers import BaseError
 
-# Global variables used to ensure that there are no duplicate node names
 list_of_lists_of_previous_cell_values: list[list[str]] = []
+"""Module level variable used to ensure that there are no duplicate node names"""
+
 list_of_previous_node_names: list[str] = []
+"""Module level variable used to ensure that there are no duplicate node names"""
+
 
+def expand_lists_from_excel(
```

Comment:
I moved this method here, because it thematically belongs here. Before, it was in the file `expand_all_lists.py`, which contained nothing else than just this method. I think it makes more sense to move it here and delete the other file.